### ISSUE TITLE
indexing existing functionality

### ISSUE THREAD

**Issue by codercahol** [TimothyStiles/poly#217]
**Is your feature request related to a problem? Please describe.**
while trying to get up to speed on poly, I'm not sure what functions already exist/how the project is organized

**Describe the solution you'd like**
an index of existing functionality (as part of one of the .md docs) - preferralby at a higher level of solutions that poly is currently solving in some way (as opposed to a detailed, constantly updating list of all the functions implemented)

**Comment by codercahol** [comment]
hadn't seen that, thanks, I'm new to Go. 

I was imagining something a little bit more abstracted.. somewhere btw an elevator pitch and the docstrings for each function

Im missing a global picture of context; ie "we are designing genes, practically this means we need a protocol to integrate the genes into genomes, practicallythis means we need to design different methods for assembling genes w overhangs/restriction enzymes etc" except better bc im still a synbio noob lol

basically, i understand how poly fits into the world and how the functions in poly do things, but i dont understand how those functions fit into poly

**Comment by TimothyStiles** [comment]
@codercahol alot of that is written in line and actually in the subpackage directories themselves. Have you had a chance to check that out?

**Comment by TimothyStiles** [comment]
I've also been updating docs a little bit too. More to come.
